### PR TITLE
chore(release): alternative headless mode release notes

### PR DESCRIPTION
**Pull request by Skn0tt**
Here's my attempt at making the headless mode release notes more assertive in tone and a little safer to skim.

### CODE REVIEW

File: docs/src/release-notes-js.md [hunk 1]
```diff
@@ -38,38 +38,48 @@ Learn more in the [aria snapshots guide](./aria-snapshots).
 - Added "previous" and "next" buttons to the HTML report to quickly switch between test cases.
 - New properties [`property: TestInfoError.cause`] and [`property: TestError.cause`] mirroring [`Error.cause`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause).
 
-### Breaking: channels `chrome`, `msedge` and similar switch to new headless
+## Breaking: `chrome` and `msedge` channels switch to new headless mode
```
Comment (dgozman) [inline docs/src/release-notes-js.md:41]: Please keep the heading levels as before - ### for breaking, #### for details.

File: docs/src/release-notes-js.md [hunk 2]
```diff
@@ -86,6 +96,8 @@ export default defineConfig({
 });
 ```
 
+See [issue #33566](https://github.com/microsoft/playwright/issues/33566) for more details on what's changing with the new headless mode.
```
Comment (dgozman) [inline docs/src/release-notes-js.md:99]: There isn't much in that issue to refer to for these brave souls!

File: docs/src/release-notes-js.md [hunk 3]
```diff
@@ -38,38 +38,48 @@ Learn more in the [aria snapshots guide](./aria-snapshots).
...
 
-If you are only running tests in headless, for example on CI, you can avoid downloading a headed version of Chromium by specifying `chromium-headless-shell` during installation.
+- `chrome`, `chrome-dev`, `chrome-beta`, or `chrome-canary`
+- `msedge`, `msedge-dev`, `msedge-beta`, or `msedge-canary`
+
+If your `playwright.config.ts` doesn't contain any `channel: '...'` options, or none of them specify one of the above channels, you're not affected by this change today.
+
+### What do I need to do?
+
+After updating to Playwright 1.50, run your test suite. If it still passes, you're good to go. If not, you will probably need to update your snapshots, and adapt some of your test code aorund PDF viewers and extensions. See [issue #33566](https://github.com/microsoft/playwright/issues/33566) for more details.
```
Comment (dgozman) [inline docs/src/release-notes-js.md:61]: Should be 1.49, not 1.50.

**Review by pavelfeldman — APPROVED** [review]

File: docs/src/release-notes-js.md (hunk 3 above)
Comment (yury-s) [inline docs/src/release-notes-js.md:61]:
```suggestion
After updating to Playwright 1.50, run your test suite. If it still passes, you're good to go. If not, you will probably need to update your snapshots, and adapt some of your test code around PDF viewers and extensions. See [issue #33566](https://github.com/microsoft/playwright/issues/33566) for more details.
```

File: docs/src/release-notes-js.md [hunk 4]
```diff
@@ -38,38 +38,48 @@ Learn more in the [aria snapshots guide](./aria-snapshots).
...
+Chromium recently introduced a new mode for [headless operation](https://developer.chrome.com/docs/chromium/headless). While Playwright was able to continue using the old, established mode in previous versions, the old mode got removed from the chromium binary and is now distributed separately.
 
 ![Chromium Headless](https://github.com/user-attachments/assets/2829e86a-dfe2-4743-a6d4-2aa65beea890)
 
-If you are using a browser channel, for example `'chrome'` or `'msedge'`, the headless mode switch **will affect you**. Most likely, you will have to update some of your tests and all of your screenshot expectations. See [issue #33566](https://github.com/microsoft/playwright/issues/33566) for more details.
+To prevent breaking changes, Playwright continues making both modes available in the standard `chromium` channel for the foreseeable future by downloading both distributables.
+Since branded browser distributables are subject to the removal of the old mode, Playwright uses the new mode with them starting with this version. This keeps your tests running, but it comes with changes in behavior that might affect your test suite.
 
-#### Chromium headless shell
+### Am I affected?
```
Comment (yury-s) [inline docs/src/release-notes-js.md:50]: I like this heading and structure, it is straight to the point.

**Review by yury-s — APPROVED** [review]

**Pull request merged**